pkgs/by-name/fi/firefox-sync-client/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "firefox-sync-client";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Mikescher";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-CNTTlBYJlQ4z6VXgi/mZpwxIAVOxJUnIL9flyvEzmXg=";
+  };
+
+  vendorHash = "sha256-Gb+4fxMBgvPB9Ki7zIwscY9l2kJ+tuE1Mc3W08YTfk8=";
+
```

Consider adding an update script
```suggestion
	passthru.updateScript = nix-update-script { };
```